Modules/Mutes.cs

```diff
@@ -72,7 +72,7 @@ public static async System.Threading.Tasks.Task<bool> MuteUserAsync(DiscordMembe
             {
                 // A DM failing to send isn't important, but let's put it in chat just so it's somewhere.
                 if (!(channel is null))
-                    await channel.SendMessageAsync($"{Program.cfgjson.Emoji.Muted} {naughtyMember.Mention} was muted for **{Warnings.TimeToPrettyFormat(muteDuration, false)}**!");
+                    await channel.SendMessageAsync($"{Program.cfgjson.Emoji.Muted} {naughtyMember.Mention} was automuted for **{Warnings.TimeToPrettyFormat(muteDuration, false)}**!");

```

Comment:
I'd prefer something less slang like "automatically muted".
Maybe could also show that it was for multiple warns by borrowing the text from the DM?

Reply:
Also I'm pretty sure modifying this text will affect all mutes? Not just automatic ones

Reply:
> Also I'm pretty sure modifying this text will affect all mutes? Not just automatic ones

I thought so too but after initiating a manual mute it seems to print the string "has been muted" as opposed to "was muted" which seems to happen [here](https://github.com/Erisa/Cliptok/blob/main/Modules/Mutes.cs#L246).
![image](https://user-images.githubusercontent.com/31840547/109519234-f47c2780-7aa2-11eb-9cf5-5c0c141ea303.png)

Reply:
This made me think though - what happens if you manually mute someone who's blocking DMs?

Reply:
Absolutely no idea, I guess it prints two messages?

Reply:
That was my thought and was why I asked :P either way I'll look at it later